index.html

```diff
@@ -0,0 +1,158 @@
+          <p>
+            FIFA is a non-profit organization that describes itself as an international governing body of association football, futsal and beach soccer. It is the highest governing body of association football.
+          </p>
+        </div>
+        <h2>2021.10.12(tue) ~ 15(fri)</h2>
+        <p class="place">@ stellanbosch, South africa</p>
+      </section>
+
+      <section class="prog">
+        <h3>Main Program</h3>
+        <hr class="line" />
+
+        <div class="prog-line">
+          <div id="prog-0" class="cards">
+            <img src="images/calendar.svg" alt="" />
+            <p class="title">Calendar</p>
+            <p>date and time are important  to everyone around the world</p>
+          </div>
+          <div id="prog-1" class="cards">
+            <img src="images/card.svg" alt="" />
+            <p class="title">Card</p>
+            <p>Represent the worth  of every player out there shows how important you are</p>
+          </div>
+          <div id="prog-2" class="cards">
+            <img src="images/folder.svg" alt="" />
+            <p class="title">folder</p>
+            <p>to save protect your info all the time for the near future</p>
+          </div>
+          <div id="prog-3" class="cards">
+            <img src="images/people.svg" alt="" />
+            <p class="title">people</p>
+            <p>communication represent the huminity of a person to the world</p>
+          </div>
+          <div id="prog-4" class="cards">
+            <img src="images/shield.svg" alt="" />
+            <p class="title">shield</p>
+            <p>protect yourself at all time surround yourself with the one who understand you </p>
+          </div>
+        </div>
+        <h3 class="linked"><a href="">SEE THE WHOLE PROGRAM</a></h3>
+        <button type="button">join sports summit program 2021</button>
+      </section>
+
+      <section class="speakers">
+        <h3>featured speakers</h3>
+        <hr class="line" />
+        <div class="speakers-lineout"></div>
+        <button id="button" type="button">More</button>
+      </section>
+
+      <section class="sponser">
+        <h3>partners</h3>
+        <hr class="line" />
+
+        <ul class="sponser-list">
+          <li class="image"><img src="images/Coca-Cola-Logo-StartupTalky--1-.jpg" alt="" /></li>
+          <li><img src="images/Budweiser-Logo-StartupTalky.jpg" alt="" /></li>
+          <li><img src="images/Qatar-Airways-Logo-StartupTalky.jpg" alt="" /></li>
+          <li class="image">
+            <img src="images/Vivo-Logo-StartupTalky--1-.jpg" class="stretch" alt="" />
+          </li>
+          <li class="image">
+            <img src="images/Wanda-Group-Logo-StartupTalky.jpg" class="stretch" alt="" />
+          </li>
+        </ul>
+      </section>
+    </main>
+    <footer>
+      <div class="footer color-w">
+        <img src="images/Capture.png" alt="" />
+        <p>2021  Chester, all rights reserved</p>
+      </div>
```

Comment:
- [ ] Kindly make sure that the footer contains a logo aligned to the left and text aligned to the right. The logo does not load, it must be a bad referencing problem.

![image](https://user-images.githubusercontent.com/4247740/164859488-9db6ab01-93e4-417a-80dc-460d3a4480a3.png)